contracts/Bridge.sol

```diff
@@ -267,6 +280,16 @@ contract Bridge is Pausable, AccessControl, SafeMath {
         _depositCounts[domainID] = nonce;
     }
 
+    /**
+        @notice Add a forwarder to be used.
+        @notice Only callable by an address that currently has the admin role.
+        @param forwarder Forwarder address to be added.
+        @param valid Decision for the specific forwarder.
+     */
+    function adminAddForwarder(address forwarder, bool valid) external onlyAdmin {
```

Comment:
Function name contradicts the logic. It doesn't always 'add' a forwarder. Perhaps should be named adminSetForwarder.

Reply:
make sense